tests/5.0/declare_variant/test_declare_variant.F90

```diff
@@ -0,0 +1,94 @@
+!===--- test_declare_variant.F90 -------------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test uses the declare variant clause to create two variants of a
+! simple base function, one for use in a parallel region, and one for use
+! in a target region. The function sets each element of an array to its
+! index. Each variant is called on a separate array and the results are
+! checked.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+MODULE variants
+  use omp_lib
+CONTAINS
+  SUBROUTINE fn(arr)
+    INTEGER,INTENT(inout) :: arr(N)
+    INTEGER :: i
+    !$omp declare variant(p_fn) match(construct = {parallel})
+    !$omp declare variant(t_fn) match(construct = {target})
+
+    DO i = 1, N
```

Comment:
Default should do something different if we want to verify that the variant was exercised.

Reply:
Good catch, should be fixed now. I also adjusted the error checking to distinguish which variant was not used as expected.